README.md

```diff
@@ -18,7 +18,7 @@ Just add the following line to the `Cargo.toml`:
 exonum-testkit = "0.1.0"
 ```
 
-[Read more...][tutorial]
+[Tutorial][tutorial]
```

Comment:
Nitpick: I wouldn't call that particular page a tutorial. There *is* a separate tutorial (exonum/exonum-doc#143) showing how to use testkit on the example of the demo cryptocurrency. So, I would change to something like "See Exonum documentation for details". A link to the actual tutorial could be added later when it's merged and published (hopefully, soon).